src/show/pages/private/Admin/view.jsx

```diff
@@ -9,52 +9,53 @@ import {
   Modal,
   TextField
 } from 'show/components';
-import {
-  addConstructor,
-  getConstructors
-} from 'process/slices/constructorsSlice';
+import { mock } from 'process/helpers';
 import { onFieldChange } from 'process/helpers';
-import { updateProps } from 'process/slices/transientSlice';
-import { addEvent, getEvents } from 'process/slices/eventsSlice';
-import { addDriver, getDrivers } from 'process/slices/driversSlice';
-const Admin = () => {
+
+const Admin = ({
+  addConstructor,
+  addDriver,
+  addEvent,
+  constructors,
+  drivers,
+  events,
+  getConstructors,
+  getDrivers,
+  getEvents,
+  transient,
+  updateProps
+}) => {
   const [createEventModal, setCreateEventModal] = useState(false);
   const [createDriverModal, setCreateDriverModal] = useState(false);
   const [createConstructorModal, setCreateConstructorModal] = useState(false);
 
-  const dispatch = useDispatch();
-  const dispatchUpdateProps = payload => dispatch(updateProps(payload));
-
-  const transient = useSelector(state => state.transient);
-  const events = useSelector(state => state.events.events);
-  const drivers = useSelector(state => state.drivers.drivers);
-  const constructors = useSelector(state => state.constructors.constructors);
+  const dispatchUpdateProps = updateProps();
```

Comment:
Delete this line, and replace `dispatchUpdateProps` with `updateProps`. that's it